static/styles.css

```diff
@@ -0,0 +1,6 @@
+3.ply-ui {
+  color: #F47425;
+}

```

Comment:
is the `3` here supposed to be `h3`? Not seeing where a `3` tag would be generated for this to apply to...